vms/evm/acp226/acp226.go

```diff
@@ -21,6 +21,10 @@ const (
 	// MaxDelayExcessDiff (Q) is the maximum change in excess per update
 	MaxDelayExcessDiff = 200
 
+	// InitialDelayExcess represents the initial (≈2000ms) delay excess.
+	// Formula: ConversionRate (2^20) * ln(2000) + 1
+	InitialDelayExcess = 7_970_124
```

Comment:
These feel like magic numbers even though we use a godoc to document them... could we show the calculation in code? i.e
```
const ConversionRate = 1 << 20 // 2^20 fixed-point scale

var InitialDelayExcess = int(float64(ConversionRate)*math.Log(2000)) + 1
```

Reply:
These values are consensus critical, so I don't think it is reasonable to perform floating point operations to generate them. (The actual number is more important than the mechanism used to derive them)